build.gradle

```diff
@@ -43,5 +45,20 @@ subprojects {
 
     tasks.check.dependsOn(javadoc)
     sourceCompatibility = 1.8
+
+    tasks.withType(JavaCompile) {
+        options.compilerArgs += ['-Werror']
+    }
+
+    plugins.withId('com.palantir.baseline-error-prone', {
+        dependencies {
+            compileOnly 'com.google.code.findbugs:jsr305'
+            annotationProcessor 'com.uber.nullaway:nullaway'
+        }
+
+        tasks.withType(JavaCompile) {
+            options.errorprone.errorproneArgs.add '-XepOpt:NullAway:AnnotatedPackages=com.palantir'
```

Comment:
This should probably be either a separate plugin, or config option in the errorprone extension. Not suggesting we change this, just musing on potential improvements.

Reply:
Yeah agreed - I think @schlosna had the same feeling when he had to add lots of boilerplate to tritium to opt-in to extra strictness

Reply:
Yeah, having a simple NullAway opt-in inside gradle-baseline would be nice. It is definitely much easier to add NullAway to projects early on than to retrofit after the fact, and does have [some false positives around mock injections for unit tests and JMH](https://github.com/palantir/tritium/search?l=Java&q=NullAway).